test/remote/gateways/remote_stripe_payment_intents_test.rb

```diff
@@ -445,6 +462,26 @@ def test_purchase_works_with_stored_credentials_without_optional_ds_transaction_
     end
   end
 
+  def test_succeeds_with_ntid_in_stored_credentials_and_separately
```

Comment:
📌  For later, but I feel like we should be sure to mention in the docs update what happens if/when `network_transaction_id` is sent in both places.